modules/claims_api/app/services/claims_api/bgs_to_lighthouse_claims_mapper_service.rb

```diff
@@ -33,26 +33,20 @@ def lighthouse_claim_only?
 
     def matched_claim
       # this claim was submitted via Lighthouse, so use the 'id' the user is most likely to know
-      {
-        id: lighthouse_claim.id,
-        type: bgs_claim[:claim_status_type],
-        date_filed: bgs_claim[:claim_dt],
-        status: bgs_claim[:phase_type],
-        end_product_code: bgs_claim[:end_product_code],
-        documents_needed: bgs_claim[:attention_needed],
-        requested_decision: map_y_n_to_boolean('filed5103_waiver_ind', bgs_claim[:filed5103_waiver_ind]),
-        development_letter_sent: map_yes_no_to_boolean('development_letter_sent', bgs_claim[:development_letter_sent]),
-        decision_letter_sent: map_yes_no_to_boolean('decision_notification_sent',
-                                                    bgs_claim[:decision_notification_sent])
-      }
+      { id: lighthouse_claim.id }.merge(shared_claim_traits)
     end
 
     def unmatched_bgs_claim
+      { id: bgs_claim[:benefit_claim_id] }.merge(shared_claim_traits)
+    end
+
+    def shared_claim_traits
       {
-        id: bgs_claim[:benefit_claim_id],
         type: bgs_claim[:claim_status_type],
-        date_filed: bgs_claim[:claim_dt],
+        date_filed: bgs_claim[:claim_dt].strftime('%D'),
```

Comment:
Something about this check is tripping up some tests, still working on figuring out why. Trying to include the safe navigation operator like so didn't seem to fix it:
```ruby
date_filed: bgs_claim[:claim_dt]&.strftime('%D'),
```